webapp/components/test-runs-query.html

```diff
@@ -12,17 +13,19 @@
      * Behaviour class for re-use of test-runs fetching behaviour.
      */
     // eslint-disable-next-line no-unused-vars
-    const TestRunsQuery = (superClass) => class extends QueryBuilder(superClass) {
+    const TestRunsQuery =
+      (superClass) => class extends QueryBuilder(ProductInfo(superClass)) {
       static get properties() {
         return {
           /* Parsed product objects, computed from the spec strings */
```

Comment:
Is this comment still true? IIUC `products` is now the parameter set by parents while `productSpecs` is the computed value.

Reply:
Actually, both; if you set the product-specs value initially, it'll tee up the `products` array from those specs.